src/storage/src/redis_pkhashes.cc

```
@@ -0,0 +1,1279 @@
+  UpdateSpecificKeyStatistics(DataType::kPKHashes, key.ToString(), statistic);
+  return s;
+}
+
+Status Redis::PKHMSet(const Slice& key, const std::vector<FieldValue>& fvs) {
+  uint32_t statistic = 0;
+  std::unordered_set<std::string> fields;
+  std::vector<FieldValue> filtered_fvs;
+  for (auto iter = fvs.rbegin(); iter != fvs.rend(); ++iter) {
+    std::string field = iter->field;
+    if (fields.find(field) == fields.end()) {
+      fields.insert(field);
```

cppcheck

[performance] Searching before insertion is not necessary.

(stlFindInsert)